contributing-guides/style-guide.md

```diff
@@ -283,7 +283,83 @@ In order to maintain readability and normalization, please comply with the 6 rul
 
 For more information and examples of Chinese-specific rules, check out [*Chinese Copywriting Guidelines*](https://github.com/sparanoid/chinese-copywriting-guidelines/blob/master/README.en.md).
 
+### Indonesian-Specific Rules
+
+When translating pages to Indonesian, please keep in mind that we expect `tldr` pages to be easy to read for **both types of Indonesian audiences**, which are:
+
+1. People who prefer to use standard Indonesian technical terms as possible, such as `unduh` for `download`, `awakutu` for `debugging`, and `muat ulang` for `reboot`.
+  - One of the most comprehensive list of technical terms can be found under the [BlankOn Linux project](https://dev.blankonlinux.or.id/TimPengembang/Dokumentasi/Panduan/PanduanWiki/KamusBlankOn/).
+2. People who prefer to use English words as-is to describe technical terms: `download` for `download`, `debugging` for `debugging`, and `reboot` for `reboot`.
+
+The segmentation of these audiences are clearly noted on [Firefox Public Data Report](https://data.firefox.com/dashboard/usage-behavior):
+
+> For most countries in the top 10, the majority (>90%) of users have their language set to the local language, **with a notable exception in Indonesia, which has about 80% English (US) and 20% Indonesian.**
+
+First, command and example descriptions on pages in Indonesian must be written **without using active verb forms (i.e. those with `ber-` and `me-` prefixes)**. This means that sentences such as:
+
+> **Mengunduh** sebuah file ke dalam suatu direktori
+> (i.e. Downloading a file into a directory)
+
+is considered incorrect. The correct form of the sentence should be:
+
+> **Unduh** sebuah file ke dalam suatu direktori
+
+If you have seen a translated page with these active verbs, it may be due to a mistake previously made by Indonesian translation maintainers. Of course, we welcome your help to fix them. See https://github.com/tldr-pages/tldr/issues/8576 for details.
```

Comment:
```suggestion
If you have seen a translated page with these active verbs, it may be due to a mistake previously made by Indonesian translation maintainers. Of course, we welcome your help to fix them. See <https://github.com/tldr-pages/tldr/issues/8576> for more details.
```

Alternatively, we can drop this whole sentence.

```suggestion
```